services/tktrex/extension/src/app.ts

```diff
@@ -189,6 +193,37 @@ function setupObserver(): void {
   }
 }
 
+const hidLog = log.extend('intercept-data-listener');
+/**
+ * handle a new intercepted datum node by dispatching
+ * the event to the hub and remove the node from the container
+ */
+
+const handleInterceptedData = (): void => {
+  const itemNodes = document.body.querySelectorAll(
```

Comment:
not sure if it is necessary, it should arrive as argument of the call

Reply:
this handler is not defined as callback for a mutation observer, but is a function invoked inside a `setInterval` and needs to get the the new element at each loop